src/index.html

```diff
@@ -1,16 +1,42 @@
 <!doctype html>
-<html lang="en">
+<html lang="ru">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="get"
+    class="form__big form__our"
+    data-qa="big">
```

Comment:
Go through [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) one more time and fix your code style, attributes should appear on a new line